### ISSUE TITLE
Error/warning during yolo26 training

### ISSUE THREAD

**Issue by xavan83** [ultralytics/ultralytics#24112]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Ultralytics YOLO Component

Train

### Bug

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size
       1/80       6.6G      1.006      3.862     0.0116         59        640: 100% ━━━━━━━━━━━━ 276/276 2.3it/s 1:59
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100% ━━━━━━━━━━━━ 1/1 1.4it/s 0.7s
**/home/xavan/miniconda3/envs/yolo313/lib/python3.13/site-packages/ultralytics/utils/metrics.py:839: RuntimeWarning: Mean of empty slice.
  i = smooth(f1_curve.mean(0), 0.1).argmax()  # max F1 index
/home/xavan/miniconda3/envs/yolo313/lib/python3.13/site-packages/numpy/_core/_methods.py:137: RuntimeWarning: invalid value encountered in divide
  ret = um.true_divide(**
                   all          1          0          0          0          0          0

### Environment

Ultralytics 8.4.33 🚀 Python-3.13.11 torch-2.10.0+cu130 CUDA:0 (NVIDIA GeForce RTX 5050, 7708MiB)
Setup complete ✅ (8 CPUs, 30.1 GB RAM, 192.3/465.4 GB disk)

OS                     Linux-6.17.0-20-generic-x86_64-with-glibc2.42
Environment            Linux
Python                 3.13.11
Install                pip
Path                   /home/xavan/miniconda3/envs/yolo313/lib/python3.13/site-packages/ultralytics
RAM                    30.14 GB
Disk                   192.3/465.4 GB
CPU                    12th Gen Intel Core(TM) i3-12100
CPU count              8
GPU                    NVIDIA GeForce RTX 5050, 7708MiB
GPU count              1
CUDA                   13.0

numpy                  ✅ 2.2.6>=1.23.0
matplotlib             ✅ 3.10.8>=3.3.0
opencv-python          ✅ 4.13.0.90>=4.6.0
pillow                 ✅ 12.0.0>=7.1.2
pyyaml                 ✅ 6.0.3>=5.3.1
requests               ✅ 2.32.5>=2.23.0
scipy                  ✅ 1.17.0>=1.4.1
torch                  ✅ 2.10.0+cu130>=1.8.0
torch                  ✅ 2.10.0+cu130!=2.4.0,>=1.8.0; sys_platform == "win32"
torchvision            ✅ 0.25.0+cu130>=0.9.0
psutil                 ✅ 7.2.1>=5.8.0
polars                 ✅ 1.37.1>=0.20.0
ultralytics-thop       ✅ 2.0.18>=2.0.18

### Minimal Reproducible Example

if __name__ == '__main__':
    import os

    from ultralytics import YOLO
    from ultralytics import models

    import ultralytics
    import torch.multiprocessing as mp

    DATA_FILE_CROPPED="cropped.yaml"
    YOLO("yolo26s.pt").train(compile=True,
                                     seed=1234,
                                     deterministic=True,
                                     rect=False,
                                     pretrained=True,
                                     amp=True,
                                     epochs=80,
                                     batch=24,
                                     project='yolo26',
                                     optimizer='MuSGD',
                                     workers=4,end2end=False,data="/home/xavan/train/cropped.yaml")

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by UltralyticsAssistant** [comment]
👋 Hello @xavan83, thank you for reporting this and for including your environment details and a reproducible training snippet 🚀 This is an automated response to help get things moving quickly, and an Ultralytics engineer will also assist you soon.

We recommend a visit to the [Docs](https://docs.ultralytics.com/) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples and where many of the most common questions may already be answered.

Since this appears to be a 🐛 Bug Report, thank you for already sharing a reproducible example 🙌 To help with debugging, please also provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) if you can further reduce it, along with any dataset structure details, a sample of your labels, and the full training/validation console output around the warning.

If this is a custom training ❓ Question, please provide as much information as possible, including dataset image examples and training logs, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo26"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by Y-T-G** [comment]
It can happen if your labels are empty. Can you check your logs before the training starts for warnings about labels?

**Comment by xavan83** [comment]
> It can happen if your labels are empty. Can you check your logs before the training starts for warnings about labels?

You are right, thank you

**Comment by glenn-jocher** [comment]
Thanks for confirming — after fixing the labels, please delete any `*.cache` files in the `labels` folder so YOLO rescans the dataset, then rerun training; if anything still looks off, please open a new issue with the full log and `yolo checks` output, and you can follow our [MRE guide](https://docs.ultralytics.com/help/minimum-reproducible-example/) to keep it easy to reproduce.
